packages/framework/block-std/src/view/utils/range-binding.ts

```diff
@@ -298,7 +298,7 @@ export class RangeBinding {
         },
         to: null,
       });
-      this.host.selection.set([selection]);
+      this.host.selection.setGroup('note', [selection]);
```

Comment:
Do we have better solution? I want to move the `note` group from std to blocks. Not sure if it's possible.

Reply:
I think it's fine, but before that, we need to improve the type of the group first, now it's all hard-coded.